### DISCUSSION TITLE
Anyone able to successfully use updateKeyAndPermissions in WhitelistOperation?

### DISCUSSION THREAD

**Discussion by itsMeDavidV** [ArchGryphon9362/teslabtapi#4] [Q&A]
For me the message goes through and I immediately get a blank response with a counter, however the key is not updated.

Wondering if anyone has successfully updated a key? 

I'm aware that you can use the old key to add another, however I want to avoid this as that can cause confusion with the user as they will get a generic push notif from Tesla that will tell them a key was added and if they do not recognize (which they wont) that they can remove it. 

It's important to be able to update the key in the unlikely, but possible event that a counter reaches the mentioned 60k limit.

**Comment by LexNastin** [comment]
> It's important to be able to update the key in the unlikely, but possible event that a counter reaches the mentioned 60k limit.

I thought that too, but haven't ever tried anything with the updateKey functions. We can wait and see if anyone else has tried it.
